### DISCUSSION TITLE
Nvidia for liver segmentation using segment from the boundary point

### DISCUSSION THREAD

**Discussion by akmal871026** [Project-MONAI/MONAI#2957] [General]
Hello,

Before, I used Nvidia for liver segmentation using segments from the boundary point. but this time there not have. Why?

Any suggestion? because clara 4.0 take over for annotation liver that using clara 2.0

**Comment by Nic-Ma** [comment]
Hi @akmal871026 ,

Thanks for your interest here.
Sorry I don't quite understand your question, could you please help share more details?

Thanks in advance.

**Comment by akmal871026** [comment]
Hi sir, can you give video how to do liver segmentation using MONAI?

Or you have the algorithm at 3D Slicer for it?

On Tue, Sep 28, 2021 at 10:35 PM Nic Ma ***@***.***> wrote:

> Hi @akmal871026 <https://github.com/akmal871026> ,
>
> Thanks for your interest here.
> Sorry I don't quite understand your question, could you please help share
> more details?
>
> Thanks in advance.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Project-MONAI/MONAI/discussions/2957#discussioncomment-1396560>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASFIPQ26YQTFXYDF2PK7CWDUEHHCPANCNFSM5E5SWCGA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 








*___________________________________*

*MOHD AKMAL BIN MASUD*

Pegawai Sains (Fizik),

No. 4, Jalan P7, Presint 7,

Jabatan Perubatan Nuklear,

Institut Kanser Negara,

62250 Putrajaya.

E-mail: ***@***.***

            ***@***.***

H/P: +60143321631

**Reply by Nic-Ma** [reply]
I think maybe you can refer to the 3D spleen segmentation tutorial: https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/spleen_segmentation_3d.ipynb

Thanks.
